spec/unit/provider/systemd_unit_spec.rb

```diff
@@ -222,33 +222,31 @@
 
     # A password is required when specifying a user on Windows. Since systemd resources
     # won't actually run on Windows, skip these tests rather than code a workaround.
-    unless windows?
-      it "loads the user unit content if the file exists and user is set" do
-        new_resource.user("joe")
-        allow(File).to receive(:exist?)
-          .with(unit_path_user)
-          .and_return(true)
-        allow(File).to receive(:read)
-          .with(unit_path_user)
-          .and_return(unit_content_string)
-        expect(File).to receive(:exist?)
-          .with(unit_path_user)
-        expect(File).to receive(:read)
-          .with(unit_path_user)
-        provider.load_current_resource
-        expect(current_resource.content).to eq(unit_content_string)
-      end
+    it "loads the user unit content if the file exists and user is set" do
+      new_resource.user("joe")
+      allow(File).to receive(:exist?)
+        .with(unit_path_user)
+        .and_return(true)
+      allow(File).to receive(:read)
+        .with(unit_path_user)
+        .and_return(unit_content_string)
+      expect(File).to receive(:exist?)
+        .with(unit_path_user)
+      expect(File).to receive(:read)
+        .with(unit_path_user)
+      provider.load_current_resource
+      expect(current_resource.content).to eq(unit_content_string)
+    end
 
-      it "does not load the user unit if the file does not exist and user is set" do
-        new_resource.user("joe")
-        allow(File).to receive(:exist?)
-          .with(unit_path_user)
-          .and_return(false)
-        expect(File).to_not receive(:read)
-          .with(unit_path_user)
-        provider.load_current_resource
-        expect(current_resource.content).to eq(nil)
-      end
+    it "does not load the user unit if the file does not exist and user is set" do
+      new_resource.user("joe")
+      allow(File).to receive(:exist?)
+        .with(unit_path_user)
+        .and_return(false)
+      expect(File).to_not receive(:read)
+        .with(unit_path_user)
+      provider.load_current_resource
+      expect(current_resource.content).to eq(nil)
```

Comment:
I changed these tests in #10127. After some discussion in slack, I modified these this way because a developer using Windows might want to run the unit tests for the systemd resource, even though systemd will realistically not run on Windows. Given that, the changes in this PR should be reverted.